hudi-client/src/main/java/org/apache/hudi/client/bootstrap/BootstrapMode.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.client.bootstrap;
+
+public enum BootstrapMode {
+  FULL_BOOTSTRAP,
+  METADATA_ONLY_BOOTSTRAP
```

Comment:
rename to 
`RECORD_DATA_BOOTSTRAP`, `RECORD_METADATA_BOOTSTRAP` (or)
just `THICK_BOOTSTRAP`, `THIN_BOOTSTRAP` that more user understandable and relatable to eachother? 
We atleast need to call it `RECORD_METADATA_ONLY_BOOTSTRAP` since now, it feels like we are just copying file metadata.. which we are not..